tools/check/ut.go

```diff
@@ -397,9 +400,23 @@ func handleFlags(flag string) string {
 	return res
 }
 
+func handleRaceFlag() (found bool) {
```

Comment:
Combine `handleFlags` and `handleRaceFlag` is better. Like add an int param for `handleFlags` to figure out how many args for this flag.

Reply:
When the params are complex enough, I'd refactor the code to introduce some more complex packages like `flags`
But I think I can still bear with the complexity.
Any way, it's just used as a quick and dirty script.